source/_patterns/02-molecules/blocks/alert/alert.yml

```diff
@@ -15,3 +15,4 @@ alert:
   heading: "Informative status"
   text: "<p>Lorem ipsum dolor sit amet, <a href=\"javascript:void(0);\">consectetur adipiscing</a> elit, sed do eiusmod.</p>"
   close: true
+  cookie_disabled: true

```

Comment:
@baptisten I don't see this used anywhere.

Reply:
Good catch... I ended up not using this.